Cargo.toml

```diff
@@ -1,29 +1,29 @@
-[workspace]
+[package]
+name = "ark-std"
+version = "0.1.0"
+authors = [ "arkworks contributors" ]
+description = "A library for no_std compatibility"
+homepage = "https://arkworks.rs"
+repository = "https://github.com/arkworks-rs/utils"
+documentation = "https://docs.rs/ark-std/"
+keywords = [ "no_std" ]
+categories = ["cryptography"]
+include = ["Cargo.toml", "src", "README.md", "LICENSE-APACHE", "LICENSE-MIT"]
+license = "MIT/Apache-2.0"
+edition = "2018"
 
-members = [
-    "bench-utils",
-    "std",
-]
+[dependencies]
+rand = { version = "0.7", default-features = false }
+# rand = { version = "0.8", default-features = false, features = ["std_rng"] }
+rand_xorshift = "0.2"
+# rand_xorshift = "0.3"
+rayon = { version = "1", optional = true }
 
-[profile.release]
```

Comment:
Don't we still want these profiles?

Reply:
Oops, you're right. Thanks for catching that.